src/ci/docker/x86_64-gnu-llvm-3.7/Dockerfile

```diff
@@ -26,5 +26,6 @@ ENTRYPOINT ["/usr/bin/dumb-init", "--"]
 
 ENV RUST_CONFIGURE_ARGS \
       --build=x86_64-unknown-linux-gnu \
-      --llvm-root=/usr/lib/llvm-3.7
+      --llvm-root=/usr/lib/llvm-3.7 \
+      --enable-test-miri
```

Comment:
> I'm not sure why we'd want to run this specific test suite out of all our test suites on Travis?

miri's test suite is currently run for a couple of architectures to make sure that e.g. both 32bit and 64bit architectures are handled correctly. And we recently learned that miri is currently broken on Mac OS (when using MIR-libstd). There's enough platform-dependent stuff going on in miri that covering more architectures is certainly desirable. Whether it is desirable enough to slow down all these workers, I cannot judge. :)